ruby-challenges.rb

```diff
@@ -31,3 +41,17 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+# def palindrome_test()
+# def palindrome_test(string)
+
+palidromes = [is_palindrome1, is_palindrome2, is_palindrome3]
+
+palidromes.each do |value|
+    puts "#{value} is a palindrome"
+  end
+unless palidromes === "LEARN"
+  puts "#{is_palindrome2} is not a palindrome"
+else
+  puts "#{value} is a palindrome"
+end
```

Comment:
Is there a way to turn this into a method that is dynamic and does not hardcode any values?  There are a lot of really helpful built-in Ruby methods that can help with this.